sql_exporter/common/crunchy_global_collector.yml

```diff
@@ -214,22 +189,52 @@ metrics:
     help: "Number of times the background writer stopped a cleaning scan because it had written too many buffers"
     values: [maxwritten_clean]
     query_ref: ccp_stat_bgwriter
-  - metric_name: ccp_stat_bgwriter_buffers_backend
-    type: gauge
-    help: "Number of buffers written directly by a backend"
-    values: [buffers_backend]
-    query_ref: ccp_stat_bgwriter
-  - metric_name: ccp_stat_bgwriter_buffers_backend_fsync
-    type: gauge
-    help: "Number of times a backend had to execute its own fsync call (normally the background writer handles those even when the backend does its own write)"
-    values: [buffers_backend_fsync]
-    query_ref: ccp_stat_bgwriter
   - metric_name: ccp_stat_bgwriter_buffers_alloc
     type: gauge
     help: "Number of buffers allocated"
     values: [buffers_alloc]
     query_ref: ccp_stat_bgwriter
 
+
+  - metric_name: ccp_stat_checkpointer_num_timed
+    type: gauge
+    help: "Number of scheduled checkpoints that have been performed"
```

Comment:
do we want to tweak the help to say this one is time-based and the other one is wal size based?

i'm good either way

Reply:
I just copied these descriptions straight from the PG docs. Rather just leave it consistent with that